libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/widthItemsHelpers.ts

```diff
@@ -246,5 +263,6 @@ export function adaptMdObjectWidthItemsToPivotTable(
         [],
         false,
         [],
+        true,
```

Comment:
in adaptReferencePointWidthItemsToPivotTable it will stay false? It is not needed there?

Reply:
For referencePoint this filter of rows/columns is already done on `adaptReferencePointWidthItemsToPivotTable` definition.